Building the docker image using `make build-docker-full` is not working on MacOS

<!--
Please use this template to create your bug report. By providing as much info as possible you help us understand the issue, reproduce it and resolve it for you quicker. Therefore take a couple of extra minutes to make sure you have provided all info needed.

PROTIP: record your screen and attach it as a gif to showcase the issue.

- Questions should be posted to: https://community.grafana.com
- Use query inspector to troubleshoot issues: https://bit.ly/2XNF6YS
- How to record and attach gif: https://bit.ly/2Mi8T6K
-->

**What happened**:
I wanted to build the docker image locally on my MacBook (m1) and it didn't workd.


**What you expected to happen**:
The make script should work on my MacBook.

**How to reproduce it (as minimally and precisely as possible)**:
Checkout the project on a MacBook with a m1 processor and run `make build-docker-full`

**Anything else we need to know?**:
The first of many errors seems to be related to chromium
```
# This file contains the result of Yarn building a package (puppeteer@npm:9.1.1)
# Script name: install

The chromium binary is not available for arm64.
If you are on Ubuntu, you can install with: 

 sudo apt install chromium


 sudo apt install chromium-browser

/grafana/.yarn/unplugged/puppeteer-npm-9.1.1-e9a012ddd7/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserFetcher.js:115
                    throw new Error();
                          ^

Error
    at /grafana/.yarn/unplugged/puppeteer-npm-9.1.1-e9a012ddd7/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserFetcher.js:115:27
    at /grafana/.pnp.cjs:57404:13
```
**Environment**:
- Grafana version: Head
- Data source type & version:
- OS Grafana is installed on: MacOS Monterey
- User OS & Browser: MacOS Monterey
- Grafana plugins:
- Others:

The chromium error seems to be gone. Only one error remains for me.

```log
 > [js-builder 13/13] RUN yarn build:
#43 52.76 [
#43 52.76   ModuleBuildError: Module build failed (from ./.yarn/__virtual__/babel-loader-virtual-bb7de44851/0/cache/babel-loader-npm-8.2.2-b4e600c2c5-df5092ef98.zip/node_modules/babel-loader/lib/index.js):
#43 52.76   SyntaxError: /grafana/public/app/index.ts: Missing semicolon. (1:7)
#43 52.76   
#43 52.76   > 1 | declare let __webpack_public_path__: string;
#43 52.76       |        ^
#43 52.76     2 | declare let __webpack_nonce__: string;
#43 52.76     3 |
#43 52.76     4 | // Check if we are hosting files on cdn and set webpack public path
#43 52.76       at Parser._raise (/grafana/.yarn/cache/@babel-parser-npm-7.15.8-4ad36dd104-a26c919676.zip/node_modules/@babel/parser/lib/index.js:541:17)
#43 52.76       at Parser.raiseWithData (/grafana/.yarn/cache/@babel-parser-npm-7.15.8-4ad36dd104-a26c919676.zip/node_modules/@babel/parser/lib/index.js:534:17)
#43 52.76       at Parser.raise (/grafana/.yarn/cache/@babel-parser-npm-7.15.8-4ad36dd104-a26c919676.zip/node_modules/@babel/parser/lib/index.js:495:17)
#43 52.76       at Parser.semicolon (/grafana/.yarn/cache/@babel-parser-npm-7.15.8-4ad36dd104-a26c919676.zip/node_modules/@babel/parser/lib/index.js:3520:10)
#43 52.76       at Parser.parseExpressionStatement (/grafana/.yarn/cache/@babel-parser-npm-7.15.8-4ad36dd104-a26c919676.zip/node_modules/@babel/parser/lib/index.js:13686:10)
#43 52.76       at Parser.parseStatementContent (/grafana/.yarn/cache/@babel-parser-npm-7.15.8-4ad36dd104-a26c919676.zip/node_modules/@babel/parser/lib/index.js:13277:19)
#43 52.76       at Parser.parseStatement (/grafana/.yarn/cache/@babel-parser-npm-7.15.8-4ad36dd104-a26c919676.zip/node_modules/@babel/parser/lib/index.js:13139:17)
#43 52.76       at Parser.parseBlockOrModuleBlockBody (/grafana/.yarn/cache/@babel-parser-npm-7.15.8-4ad36dd104-a26c919676.zip/node_modules/@babel/parser/lib/index.js:13728:25)
#43 52.76       at Parser.parseBlockBody (/grafana/.yarn/cache/@babel-parser-npm-7.15.8-4ad36dd104-a26c919676.zip/node_modules/@babel/parser/lib/index.js:13719:10)
#43 52.76       at Parser.parseProgram (/grafana/.yarn/cache/@babel-parser-npm-7.15.8-4ad36dd104-a26c919676.zip/node_modules/@babel/parser/lib/index.js:13061:10)
#43 52.76       at processResult (/grafana/.yarn/__virtual__/webpack-virtual-0eaea73f47/0/cache/webpack-npm-5.63.0-1a7a135265-8127be3d90.zip/node_modules/webpack/lib/NormalModule.js:751:19)
#43 52.76       at /grafana/.yarn/__virtual__/webpack-virtual-0eaea73f47/0/cache/webpack-npm-5.63.0-1a7a135265-8127be3d90.zip/node_modules/webpack/lib/NormalModule.js:853:5
#43 52.76       at /grafana/.yarn/cache/loader-runner-npm-4.2.0-427f0e7134-e61aea8b69.zip/node_modules/loader-runner/lib/LoaderRunner.js:399:11
#43 52.76       at /grafana/.yarn/cache/loader-runner-npm-4.2.0-427f0e7134-e61aea8b69.zip/node_modules/loader-runner/lib/LoaderRunner.js:251:18
#43 52.76       at context.callback (/grafana/.yarn/cache/loader-runner-npm-4.2.0-427f0e7134-e61aea8b69.zip/node_modules/loader-runner/lib/LoaderRunner.js:124:13)
#43 52.76       at /grafana/.yarn/__virtual__/babel-loader-virtual-bb7de44851/0/cache/babel-loader-npm-8.2.2-b4e600c2c5-df5092ef98.zip/node_modules/babel-loader/lib/index.js:59:103
#43 52.76 ]
------
executor failed running [/bin/sh -c yarn build]: exit code: 1
make: *** [build-docker-full] Error 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,7 +11,7 @@ COPY plugins-bundled plugins-bundled
 
 RUN yarn install
 
-COPY tsconfig.json .eslintrc .editorconfig .browserslistrc .prettierrc.js ./
+COPY tsconfig.json .eslintrc .editorconfig .browserslistrc .prettierrc.js babel.config.json ./
 COPY public public
 COPY tools tools
 COPY scripts scripts

--- a/Dockerfile.ubuntu
+++ b/Dockerfile.ubuntu
@@ -12,7 +12,7 @@ COPY plugins-bundled plugins-bundled
 
 RUN yarn install
 
-COPY tsconfig.json .eslintrc .editorconfig .browserslistrc .prettierrc.js ./
+COPY tsconfig.json .eslintrc .editorconfig .browserslistrc .prettierrc.js babel.config.json ./
 COPY public public
 COPY tools tools
 COPY scripts scripts

